networkx/algorithms/community/nodal_roles_nx.py

```diff
@@ -0,0 +1,128 @@
+from networkx.algorithms.community import louvain
+import networkx as nx
+
+def convert_dictionary(dictionary):
+	'''
+	Converts dictionary with nodes as keys to modules as keys
+
+	------
+	Inputs
+	------
+	dictionary =  dictionary output from Louvain community detection
+
+	------
+	Output
+	------
+	Dictionary with modules as keys, nodes as values
+
+	'''
+
+	new_dict = {}
+	for m,n in zip(dictionary.values(),dictionary.keys()):
+	    try:
+	        new_dict[m].append(n)
+	    except KeyError:
+	        new_dict[m] = [n]
+	return new_dict
+
+def get_within_module_edges(module, graph):
+	module_edges = []
+	for edge in graph.edges():
+		if edge[0] in module and edge[1] in module:
+			module_edges.append(edge)
+	return module_edges
```

Comment:
Simpler: `return [(u, v) for (u, v) in graph.edges() if u in module and v in module]`.